app/src/main/java/atlant/moviesapp/activity/EpisodeActivity.java

```diff
@@ -71,40 +77,45 @@ public void onClick(View view) {
                 onBackPressed();
             }
         });
-
+        boolean isConnected = isNetworkAvailable();
         Intent intent = getIntent();
-        Episode episode = intent.getParcelableExtra("episode");
-        Integer seriesId=intent.getIntExtra("seriesid",0);
-        Integer seasonId=intent.getIntExtra("seasonid",0);
-
+        Episode episode = intent.getParcelableExtra(getString(R.string.episodeIntent));
+        Integer seriesId=intent.getIntExtra(getString(R.string.seriesId),0);
+        Integer seasonId=intent.getIntExtra(getString(R.string.seasonId),0);
+        episodeId=episode.getId();
         date=new Date(this);
         String year=episode.getAirDate();
-        title.setText(episode.getName()+" ("+year.substring(0, Math.min(year.length(), 4))+")");
+        title.setText(stringUtils.getTitle(episode.getName(),year));
```

Comment:
As to minimize my comments regarding the StringUtils class, please read the following and change for all the usages of the StringUtils class or any other Utility class that you create in the future.
A Utility class will have all static methods, and you will name them accordingly to your needs.
A utility class will never be instantiated, thus an object of that type of class will never exist.
Please put a private empty constructor in every Utility class that you create to prevent creating objects of that type of class. Please use only static methods from any utility class